pkg/seven/utils/types.go

```diff
@@ -13,3 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 package utils
+
+import "errors"
+
+var NotFound = errors.New("NOT FOUND")
```

Comment:
Should add comments for each export-able variable.

Reply:
Done